src/components/Quote.js

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from 'react';
+import '../App.css';
+
+function Quote() {
+  const [data, setData] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const category = 'happiness';
+    const url = `https://api.api-ninjas.com/v1/quotes?category=${category}`;
+    setIsLoading(true);
+
+    fetch(url, {
+      method: 'GET',
+      headers: {
+        'X-Api-Key': 'PkqVAH5d4n204SaOGxDg5g==du09bh3RLTEmWABS',
+        'Content-Type': 'application/json',
+      },
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        setData(data[0]);
+        setIsLoading(false);
+      })
+      .catch(() => {
+        setHasError(true);
+        setIsLoading(false);
+      });
+  }, [setData]);
+
+  if (hasError) return <div>Something went wrong!</div>;
+  if (isLoading) return <div>Loading Quote...</div>;
+
+  return (
+    <div className="quote-container">
+      <p>{data.quote}</p>
+      <p>{`By: ${data.author}`}</p>
+    </div>
+  );
+}
+
+export default Quote;
```

Comment:
- [ ] As required for the project and for good structure of your files, kindly make sure you have a main component of your application(e.g.: `src/App.js`) where you should import your two components `<Calculator />` and `<Quotes />`, then the main component `App` should be imported in `src/index.js` file. Here is an example:
```JavaScript
// src/App.js

import Calculator from "./components/Calculator"
import Quotes from "./components/Quotes"

function App() {
  return (
    <>
      <Calculator />
      <Quotes />
    </>
  )
}
// -------------------------------------------------------------------
// src/index.js file
import React from 'react';
import ReactDOM from 'react-dom/client';
import Calculate from './App';

const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
);
```
export default App
```